site/docs/messaging/campaign-management/csp/brand-vetting-api.mdx

```diff
@@ -0,0 +1,368 @@
+
+## Request Brand Re-vet
+
+#### Request URL
+<Highlight color="#079CEE">PUT</Highlight> <code>https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/brands/{brandId}/revet</code>
+
+| Request Body               | Mandatory | Description                                                                                                                  |
+|:---------------------------|:----------|:-----------------------------------------------------------------------------------------------------------------------------|
+| NONE                       | N/A       | 	This request takes no request body                                                                                     |
+
+
+#### Request Authentication
+
+The [brands](./) resource is authenticated with your [API Credentials for "Number & Account Management"](../../../account/credentials)
+
+### PUT brand re-vet
+
+```http
+PUT https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/brands/{brandId}/revet HTTP/1.1
+Content-Type: application/xml; charset=utf-8
+Authorization: Basic dXNlcm5hbWU6cGFzc3dvcmQ=
+```
+
+### Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/xml
+Location: https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/brands/{brandId}/revet
+
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<BrandRevetResponse>
+    <Brand>
+      <BrandId>BJDHM3</BrandId>
+      <CspId>CMHSJ9</CspId>
+      <EntityType>NON_PROFIT</EntityType>
+      <AltBusinessId>111111111</AltBusinessId>
+      <AltBusinessIdType>DUNS</AltBusinessIdType>
+      <BrandRelationship>MEDIUM_ACCOUNT</BrandRelationship>
+      <City>Raleigh</City>
+      <CompanyName>Bandwidth</CompanyName>
+      <Country>US</Country>
+      <DisplayName>Bandwidth</DisplayName>
+      <Ein>111111111</Ein>
+      <UniversalEin>111111111</UniversalEin>
+      <Email>Test1@bandwidth.com</Email>
+      <Phone>+18009999999</Phone>
+      <PostalCode>27606</PostalCode>
+      <State>NC</State>
+      <Street>1200 Test Road</Street>
+      <StockExchange>NASDAQ</StockExchange>
+      <StockSymbol>BAND</StockSymbol>
+      <Vertical>COMMUNICATION</Vertical>
+      <Website>https://www.bandwidth.com</Website>
+      <IsMain>true</IsMain>
+    </Brand>
+</BrandRevetResponse>
+```
+
+### Error Response
+
+```http
+HTTP/1.1 400 Bad Request
+Content-Type: application/xml
+Location: https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/brands/{brandId}/revet
+
+<?xml version="1.0" encoding="UTF-8"?>
+<BrandRevetResponse>
+   <ResponseStatus>
+      <ErrorCode>12210</ErrorCode>
+      <Description>The Brand 'BEAQBHG'  with identity status 'VETTED_VERIFIED' not allowed to revet</Description>
+   </ResponseStatus>
+</BrandRevetResponse>
+```
+
+### Error Codes
+```http
+HTTP/1.1 400 Bad Request
+HTTP/1.1 403 Unauthorized
+HTTP/1.1 429 Too Many Requests
+```
```

Comment:
These same codes are repeated throughout and are pretty standard - i dont think we really need to highlight them here - maybe in a top or bottom level `Errors` section?